### PR TITLE
feat: Database for geocoding pipeline

### PR DESCRIPTION
**Pull request by bo0tzz**

### CODE REVIEW

File: kubernetes/apps/pipelines/geocoding/database/app/database.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  enableSuperuserAccess: true
+  superuserSecret:
+    name: geocoding-database-superuser
+  bootstrap:
+    initdb:
+      postInitTemplateSQL:
+        - CREATE EXTENSION postgis;
+        - CREATE EXTENSION postgis_topology;
+        - CREATE EXTENSION fuzzystrmatch;
+        - CREATE EXTENSION postgis_tiger_geocoder;
```
Comment (bo0tzz) [inline kubernetes/apps/pipelines/geocoding/database/app/database.yaml:17]: Arbitrary selection; feel free to adjust

File: kubernetes/apps/pipelines/geocoding/database/app/database.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      checkpoint_timeout: "300min"
+      max_wal_senders: "0"
+      wal_level: "minimal"
+      max_wal_size: "10GB"
+      checkpoint_completion_target: "0.9"
+      random_page_cost: "1.0"
+      full_page_writes: "off"
+      fsync: "off"
+  storage:
+    size: 100Gi
```
Comment (bo0tzz) [inline kubernetes/apps/pipelines/geocoding/database/app/database.yaml:32]: Is this enough?

Comment (zackpollard) [reply kubernetes/apps/pipelines/geocoding/database/app/database.yaml:32]: Almost certainly not :laughing: But will be enough for now

File: kubernetes/apps/pipelines/geocoding/database/app/secret.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: onepassword.com/v1
+kind: OnePasswordItem
+metadata:
+  name: geocoding-database-superuser
+  namespace: geocoding
+type: kubernetes.io/basic-auth
+spec:
+  itemPath: "vaults/Kubernetes/items/geocoding-database-superuser" # TODO: Generate secret in 1PW
```
[the remark this answers is not in the document]
Comment (zackpollard) [reply kubernetes/apps/pipelines/geocoding/database/app/secret.yaml:8]: Create it manually for now

**Review by zackpollard — APPROVED** [review]

**Pull request merged**